herd/libdir/aarch64deps.cat

```diff
@@ -46,7 +46,10 @@ catdep (* This option tells that the cat file computes dependencies *)
 include "aarch64util.cat"
 
 (* Local write successor *)
-let lws = [M]; (po & same-loc); [W | MMU & FAULT]
+let lws =
+ [Exp & M | Imp & Tag & R]; (po & same-loc); [Exp & W]
```

Comment:
I am not aware of arguments either way. I conjecture that the switch from `po-loc` to `po & same-loc` happened before the release of the K.a, because the latter phrasing was found to be easier to transliterate automatically. Perhaps, the latter should be preferred then?

Reply:
yes that makes sense, thanks!